starter-code/main.js

```diff
@@ -0,0 +1,75 @@
+const Message = {
```

Comment:
We prefer naming components with at least two words. In our case it would be something like `BulmaMessage`.